packages/react-component-playground/src/components/welcome-screen.less

```diff
@@ -0,0 +1,112 @@
+@import 'common.less';
+
+
+@clouds-texture: url('../static/clouds.png');
+@twinkling-texture: url('../static/twinkling.png');
+@stars-texture: url('../static/stars.png');
+
+@anchor-color: #08c;
+@anchor-color-focus: #005580;
+@text-color: #545454;
+
+.welcome-screen {
+  position: absolute;
+  left: @left-nav-width;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &-inner {
+    flex: .5;
```

Comment:
Flexbox has been supported in Safari since version 9, I don't get why this isn't working 😢